lib/kwtool_web/templates/auth/sign_up.html.eex

```diff
@@ -0,0 +1,17 @@
+<form class="form-authentication">
+    <h1 class="h3 mb-3 font-weight-normal">Registration</h1>
+    <label for="email">Email</label>
+    <input type="email" id="email" class="form-control mb-3" placeholder="Email Address" required autofocus>
```

Comment:
As a best practice, the team convention is to limit using the utilities of the CSS framework directly onto the HTML code. It's better to use the SCSS mixins in the component styles for loose coupling:

```
.form-authentication .form-control {
  @include make-col-offset(3);
}
```

Here are some examples: https://gist.github.com/anschaef/d7552885c0e1f127cf8830d3bbf6e4b1

Reply:
Removed the utilities, and use the 'form-group' class to solve the padding of the input elements. 

Demo:
<img width="977" alt="Screen Shot 2021-03-29 at 3 51 30 PM" src="https://user-images.githubusercontent.com/948856/112811820-a741a480-90a6-11eb-91dd-943f0aad3004.png">

I will check the SCSS mixins and use it next time.